.nvmrc

```diff
@@ -1 +1 @@
-v12.3.1
+v10.16.0
```

Comment:
Is this version ok for v2?

Reply:
This is ok, minimum is `10.9.0` according to Angular's package.json file.